spec/factories/users.rb

```diff
@@ -0,0 +1,23 @@
+FactoryGirl.define do
+  factory :user do
+  #   encrypted_password
+  #   reset_password_token
+  #   reset_passowrd_sent_at
+  #
+  #   sequence(:first_name) { |n| "first#{n}"}
+  #   sequence(:last_name) {|n| "last#{n}"}
+  #   admin false
+  sequence(:email) {|n| "hello#{n}@world.com"}
```

Comment:
Use 2 (not -1) spaces for indentation.<br>Space between { and | missing.<br>Space missing inside }.